### ISSUE TITLE
Integrating snippets

### ISSUE THREAD

**Issue by mbland** [18F/abb#6]
Since this seems to be absorbing [18F/tock](https://github.com/18F/tock), I'll submit my request here: @aaronsnow asked me to talk to @seanherron about integrating snippets functionality into time tracking.  The idea being, on one level, more granular time tracking becomes a high-level form of snippets; on the other, we can attach snippets to time-tracking items to integrate the processes.

So, the ask is, adding a snippets feature and an API so's we can import them into the [Hub](https://github.com/18F/hub).

**Comment by vdavez** [comment]
Interesting. Currently, they're just captured on Google Docs, right?

**Comment by mbland** [comment]
Yes, and it's currently me manually pinging everyone to fill out the form, then me manually harvesting the responses in a CSV file and adding them to the Hub. There's also an option to make your snippets public (they're private by default) and a syntax for redacting bits from public snippets (between `{{` and `}}` tokens).
